src/components/layout/Sidebar.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState, useContext } from 'react';
+import { Logo } from '../icons/Logo';
+import SideBarButton from '../atoms/SideBarButton/SideBarButton';
+import buttonData from '../atoms/SideBarButton/Data';
+import { NavContext } from '../../context/navContext/Navcontext';
+
+interface props {
```

Comment:
This naming convention is not correct. What you should do is since its for SideBar component, the name of this interface can be SidebarProps